blocks/scripts.js

```diff
@@ -95,6 +95,7 @@ class Script extends Stack {
     super.remove(component)
     if (!this.components.length) {
       this.removeFromWorkspace()
+      this.destroy()
```

Comment:
I'm not sure if this is a good idea. On one hand, the Script is pretty much useless once the last component is removed. However, it's probably better to manually destroy it

Also, it seems pretty dumb that Scripts self destruct when they lose all their children; maybe that should be removed too since it has required slightly weirder code for branch blocks not destroying their Script while moving the blocks inside it

Reply:
meh lol it'll warn if it double destroys